README.md

```diff
@@ -28,31 +28,35 @@ One or more of the following processors:
 
 - 2080 MiB
 
+### Runtime linux requirements
+
+On Linux platforms with hybrid Nvidia GPU setup please use Nvidia driver R525 or newer. Older ones are known to have compatibility issues. Non hybrid cards are confirmed to be working with R520 and older versions.
```

Comment:
What does hybrid mean? I have no idea

Reply:
That's actually a common statement and the technical term for integrated & discrete GPU.